app/views/DR/HomeScreen/index.js

```diff
@@ -164,6 +164,10 @@ class HomeScreen extends Component {
       state: { confirmed, deaths, recovered, current, refreshing, notified },
     } = this;
 
+    const getUrlMap = () => {
+      const url = 'https://coronavirusrd.gob.do/casos-de-covid-19-confirmados/';
```

Comment:
I think this variable should have a better descriptive name, something like COVID_CASES_URL or CONFIRMED_COVID_CASES_URL
```suggestion
      const CONFIRMED_COVID_CASES_URL = 'https://coronavirusrd.gob.do/casos-de-covid-19-confirmados/';
```